src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -77,24 +77,41 @@ public void setPerson(Person target, Person editedPerson) {
      *
      * @param target       person to be removed.
      * @param editedPerson person to be added.
-     * @param order        order to be removed.
+     * @param order        order to be added.
      */
-    public void setPersonAndDeleteOrder(Person target, Person editedPerson, Order order) {
+    public void setPersonAndAddOrder(Person target, Person editedPerson, Order order) {
         setPerson(target, editedPerson);
-        internalOrderList.remove(order);
+        internalOrderList.add(order);
     }
 
-
     /**
      * Replaces the person {@code target} in the list with {@code editedPerson}.
      *
      * @param target       person to be removed.
      * @param editedPerson person to be added.
-     * @param order        order to be added.
+     * @param order        order to be removed.
      */
-    public void setPersonAndAddOrder(Person target, Person editedPerson, Order order) {
+    public void setPersonAndDeleteOrder(Person target, Person editedPerson, Order order) {
         setPerson(target, editedPerson);
-        internalOrderList.add(order);
+        internalOrderList.remove(order);
+    }
+
```

Comment:
Did you swap the order of the commands or is this just GitHub being funny hahaha

Reply:
i think github just changed it, but shldnt matter too much right HAHAH